sas-viya/metadata.json

```diff
@@ -5,6 +5,9 @@
   "description": "The SAS Viya Quickstart Template for Azure deploys these products on the cloud: SAS Visual Analytics 8.3.1 on Linux. SAS Visual Statistics 8.3.1 on Linux and SAS Visual Data Mining and Machine Learning 8.3.1 on Linux. This Quickstart is a reference architecture for users who want to deploy the SAS Viya platform, using microservices and other cloud-friendly technologies. By deploying the SAS platform on Azure, you get SAS analytics, data visualization, and machine learning capabilities in an Azure-validated environment. SAS Viya is a cloud-enabled, in-memory analytics engine. It uses elastic, scalable, and fault-tolerant processing to address complex analytical challenges. SAS Viya provides faster processing for analytics by using a standardized code base that supports programming in SAS, Python, R, Java, and Lua. It also supports cloud, on-premises, or hybrid environments and deploys seamlessly to any infrastructure or application ecosystem.",
   "summary": "This Quickstart is a reference architecture for users who want to deploy the SAS Viya platform, using microservices and other cloud-friendly technologies.",
   "githubUsername": "sassoftware",
-  "dateUpdated": "2019-12-04"
+  "dateUpdated": "2020-02-06",
+  "environments": [
+    "AzureCloud"
```

Comment:
this doesn't work in Azure Gov?  (I thought it did)

Reply:
We do not have any resources that should disqualify us from us gov, but I do not have a test environment with access to US Gov. In running against your automated test, we ran into issues where the dns of the storage account is different from the standard. While this is a simple enough fix, without a test environment we cannot run this through QA, so for now I have to limit our support to the general cloud only.

Reply:
ok, lmk if you want to revisit or need help debugging...